ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/recovery/RecoverySuspendTest.java

```diff
@@ -0,0 +1,172 @@
+package com.hp.mwtests.ts.arjuna.recovery;
```

Comment:
I think you should add the copyright header like you did in ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/resources/BytemanControlledRecord.java

Reply:
Done. Thanks Tom